libraries/stdlib/src/kotlin/text/Char.kt

```diff
@@ -8,6 +8,102 @@
 
 package kotlin.text
 
+
+/**
+ * Creates a Char with the specified [code], or throws an exception if the [code] is out of `Char.MIN_VALUE.code..Char.MAX_VALUE.code`.
+ *
+ * @sample samples.text.Chars.charFromCode
+ */
+@ExperimentalStdlibApi
+@SinceKotlin("1.4")
+@kotlin.internal.LowPriorityInOverloadResolution // to not clash with Char constructor in js-ir
```

Comment:
Why does it help? Is it because these functions are in other package than the Char itself?